### PR TITLE
Add new gulcalc alloc rule where total peril loss = maximum subperil loss

### PR DESCRIPTION
**Pull request by hchagani-oasislmf**
New alloc rule assigned number 2. This can be changed if desired. Please supply argument `-a2` to try it out.

### CODE REVIEW

**Comment by johcarter** [comment]
looks good
